specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```
@@ -0,0 +1,2448 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v3.1-preview.4",
+    "contact": {
+      "name": "Microsoft Cognitive Services",
+      "url": "https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/",
+      "email": "mlapi@microsoft.com"
+    },
+    "title": "Text Analytics Client",
```

It will be great if we can have a title to represent the version of the TA API instead of calling it a client. It helps in APIM import.

```suggestion
    "title": "Text Analytics API (v3.1-preview.4)",
```

---

I can change this, but would this have any other possible negative impacts? Also should we then change it for all the other versions titles? @suhas92 @assafi

---

I don't think so it has any negative impact as such. I usually do change this once I import it in APIM.